includes/agentic-commerce/class-wc-stripe-agentic-commerce-manual-approval.php

```
@@ -84,8 +84,18 @@ public function validate( WC_Stripe_Agentic_Customize_Checkout_Event $event ): a
 	 * @throws Exception When product resolution fails.
 	 */
 	private function validate_line_item( WC_Stripe_Agentic_Customize_Checkout_Line_Item $line_item ): ?array {
-		$product_id = (int) $line_item->get_sku_id();
-		$product    = WC_Stripe_Agentic_Commerce_Product_Resolver::resolve_product( $product_id );
+		$product_id = wc_get_product_id_by_sku( $line_item->get_sku_id() );
+		if ( ! $product_id ) {
+			throw new Exception(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '89', column '14'). (undefined)

(MissingImport)